endpoint.go

```diff
@@ -777,7 +766,7 @@ func (ep *endpoint) Delete(force bool) error {
 
 	ep.releaseAddress()
 
-	return nil
+	return n.getEpCnt().DecEndpointCnt()
```

Comment:
Yes, but the caller does not have enough info to decide whether the error can be discarded.
In this specific case, the endpoint.Delete() function itself already knows this error can be discarded, because it knows the driver was succesfull in removing the ep and that the IP was also released. That's why I do not see a reason for propagating this internal error upstream.